components/profile-card.tsx

```diff
@@ -0,0 +1,157 @@
+import { Avatar, AvatarFallback, AvatarImage } from "./ui/avatar";
+import { Button } from "./ui/button";
+import { BadgeCheck, CalendarDays, MapPin } from "lucide-react";
+import Link from "next/link";
+import React from "react";
+
+interface ProfileCardProps {
+  name: string;
+  username: string;
+  bio: string;
+  location: string;
+  following: number;
+  followers: number;
+  coverImage: string;
+  profileImage: string;
+  createdAt: Date;
+  verified?: boolean;
+  isOwner: boolean;
+  isFollowing: boolean;
+}
+
+const ProfileCard = ({
+  name,
+  username,
+  bio,
+  location,
+  following,
+  followers,
+  coverImage,
+  profileImage,
+  createdAt,
+  verified = true,
+  isOwner,
+  isFollowing,
+}: ProfileCardProps) => {
+  const joinDate = `Joined ${createdAt.toLocaleString("default", { month: "long" })} ${createdAt.getFullYear()}`;
+
+  return (
+    <div className="h-fit">
+      <div className="relative">
+        <div
+          className="h-[200px] w-full bg-cover bg-center"
+          style={{ backgroundImage: `url(${coverImage})` }}
+        ></div>
+
+        <div className="absolute top-[140px] pl-5">
+          <Avatar className="h-[120px] w-[120px] rounded-full border-4 border-black object-cover">
+            <AvatarImage src={profileImage} alt="Profile Photo" />
+            <AvatarFallback>CN</AvatarFallback>
+          </Avatar>
+        </div>
+      </div>
+
+      <div className="px-5">
+        <div className="flex flex-col justify-start gap-3.5 pt-[70px]">
+          <div className="flex items-center justify-between">
+            <div>
+              <span className="flex flex-row items-center gap-2 text-base font-bold">
+                <p className="truncate">{name}</p>
+                {verified && (
+                  <BadgeCheck className="size-5 flex-none stroke-sky-400" />
+                )}
+              </span>
+              <p id="username" className="text-username text-sm">
+                @{username}
+              </p>
+            </div>
+
+            <div className="flex gap-4">
```

Comment:
```suggestion
          <div className="flex items-center justify-between gap-3">
            <div>
              <span className="flex flex-row items-center gap-2 text-base font-bold">
                <p className="truncate">{name}</p>
                {verified && (
                  <BadgeCheck className="size-5 flex-none stroke-sky-400" />
                )}
              </span>
              <p id="username" className="text-username text-sm">
                @{username}
              </p>
            </div>

            <div className="flex">
```
The gap should go inbetween the button and username not within the button.